Makefile.build

```diff
@@ -359,11 +364,15 @@ GRAMC := $(OCAMLFIND) ocamlc $(CAMLFLAGS) $(CAMLDEBUG) $(USERFLAGS) \
 grammar/grammar.cma : $(GRAMCMO)
 	$(SHOW)'Testing $@'
 	@touch grammar/test.mlp
-	$(HIDE)$(GRAMC) -pp '$(CAMLP5O) -I $(MYCAMLP5LIB) $^ -impl' -impl grammar/test.mlp -o grammar/test
+	$(HIDE)$(GRAMC) -pp '$(CAMLP5O) $^ -impl' -impl grammar/test.mlp -o grammar/test
 	@rm -f grammar/test.* grammar/test
 	$(SHOW)'OCAMLC -a $@'
 	$(HIDE)$(GRAMC) $^ -linkall -a -o $@
 
+grammar/gramext: $(GRAMEXTCMO) grammar/gramext_main.ml
+	$(SHOW)'OCAMLC -a $@'
+	$(HIDE)$(GRAMC) $^ -linkall -o $@
```

Comment:
This binary will have to be installed, so I propose you use here the final name, which should have coq on it. In the Dune build we already build an optimized preprocessor, named `coqp5`, maybe you could reuse that name?

Reply:
Technically no plugin rely on it yet, but it is probably going to be the case indeed. I don't have anything against calling it `camlp5`. Do other people agree?

Reply:
I prefer `coqp4`. This is not even a troll :)

Reply:
What does `coqp5` mean?

Reply:
coqppx?
camlpcoq?

Reply:
> What does coqp5 mean?

An ancient african word that means *I do not want Coq to depend on camlp5*?

Reply:
If anything, `coqpp` would be closer to reality.

Reply:
I like `coqpp`.